torch/onnx/__init__.py

```diff
@@ -46,7 +46,6 @@
 __all__ = [
     # Modules
     "symbolic_helper",
-    "symbolic_registry",
```

Comment:
Did we expose registry for a reason?

Reply:
Older code in ort manipulates the data structure in `symbolic_registry` directly for unregistering. It was very hacky and error prone and we should no longer support that. ort now conditionally uses the public apis in torch for function registration.